index.html

```diff
@@ -429,5 +429,36 @@ <h2 class="about-title">
 				</li>
 			</ul>
 		</section>
+
+		<section class="contact">
+			<div class="contact-container">
+				<div class="contact-header">
+					<h2>Contact me</h2>
+					<p>
+						If you have an application you are interested in developing, a
+						feature that you need built or a project that needs coding. I’d love
+						to help with it
+					</p>
+				</div>
+				<form action="https://formspree.io/f/xzbygpgy" method="POST">
+					<ul class="form-container">
+						<li class="form-element">
+							<input name="name_input" type="text" minlength="10" maxlength="30" placeholder="Name" required class="form-input" />
+						</li>
+						<li class="form-element">
+							<input name="email_input" type="email" minlength="10" maxlength="30" placeholder="Email" required class="form-input" />
+						</li>
+						<li class="form-element">
+							<textarea name="message_textarea" minlength="10" maxlength="500" cols="30" rows="7" required class="form-input" placeholder="Write your message"></textarea>
+						</li>
+						<li class="form-element-button">
+							<button class="bg-white-button" type="submit">
+								Get It Touch
+							</button>
+						</li>
+					</ul>
+				</form>
+			</div>
+		</section>
```

Comment:
- [ ] Kindly remove the extra indentations from here too.

_For more details, see the comment above._